pkg/commands/artifact/sbom.go

```diff
@@ -0,0 +1,80 @@
+package artifact
+
+import (
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/types"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+)
+
+type ArtifactType string
+
+const (
+	DockerImageArtifact   ArtifactType = "docker"
```

Comment:
Trivy also supports Podman and we're trying to add support for Containerd. How about `image`?

```suggestion
	ContainerImageArtifact   ArtifactType = "image"
```